dokku

```diff
@@ -120,7 +120,7 @@ case "$1" in
         exec >/dev/null 2>/dev/null </dev/null
         trap '' INT HUP
         sleep $WAIT
-        docker kill $oldid
+        docker stop $oldid
```

Comment:
In practice this does not work consistently. See the open PR for scaling. (I'm on mobile. Can't remember the number) 

Instead I think we want to attempt `stop` (default timeout is 10 sec) then `kill`.